src/IndexingWrapper.cpp

```diff
@@ -137,6 +138,7 @@ indexWrapper(Rcpp::CharacterVector & dbpath, Rcpp::DataFrame & dataFrame, Rcpp::
     if (id < 0 || id > dfCols) Rcpp::stop("Invalid argument for indexCol");
     Rcpp::List indexList = indexFields;
     Rcpp::List filterList = filterFields;
+    Rcpp::List slotList = valueSlots;
```

Comment:
Is there a reason to do this? I'd have thought you could operate directly on `valueSlots`. (This may be true for some of the other parameters as well.)